qcodes/dataset/data_export.py

```diff
@@ -10,6 +11,14 @@
 log = logging.getLogger(__name__)
 
 
+class DSPlotData(TypedDict):
```

Comment:
Does the DS stand for data structure?

Reply:
DataSet I think. This is by all means not a great name so if you have some better suggestion happy to change it

Reply:
hmm spelling out DataSetPlotData does seem a bit much. Maybe a simple doc string for clarity is god enough?